packages/net/http/_server_conn_handler.pony

```diff
@@ -90,3 +90,6 @@ class _ServerConnHandler is TCPConnectionNotify
     try
       (_session as _ServerConnection).cancel(Payload.request())
     end
+
+  fun ref connect_failed(conn: TCPConnection ref) =>
+    None
```

Comment:
as per your earlier comment:

> When a TCPConnectionNotify is being used on the server-side (as the notifier for the server-side representations of client connections, it is impossible for connect_failed to ever be called, so it seems silly to have to implement the method. I wonder if in the long term we need to use a different interface for the server-side notifier?

I wonder how we could handler `connect_failed` on server side?